jenkinsfile-runner-steward-image/Dockerfile

```diff
@@ -1,7 +1,18 @@
-FROM jenkinsfile-runner-base-local
+FROM jenkinsfile-runner-base-local as packager-output
+FROM openjdk:8-alpine
 
-RUN apk add jq python3 xmlstarlet
+RUN apk add jq python3 xmlstarlet bash git
+RUN addgroup -g 1000 jenkins && adduser -D -u 1000 -G jenkins jenkins
+RUN mkdir -p /run && chmod o+w /run
+RUN mkdir /jenkins_home && chown jenkins:jenkins /jenkins_home
+USER jenkins:jenkins
 
-COPY scripts/ /scripts
+COPY --from=packager-output --chown=jenkins:jenkins /app /app
+COPY --from=packager-output --chown=jenkins:jenkins /usr/share/jenkins /usr/share/jenkins
+
+# we want to clone the pipeline directory here:
+RUN mkdir /tmp/pipeline
+WORKDIR /tmp/pipeline
```

Comment:
why not `/workspace` as before?

Reply:
I didn't figure out what "before" was. I will change it to "workspace".